test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -43,6 +43,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			return;
 		}
 		await this.driver.switchTo().defaultContent();
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, this.editoriFrameSelector );
```

Comment:
This change makes sure the editor is there before we try to interact with it. We were sometimes getting intermittent failures saying that the element was no longer there. This change seems to fix that.